File: PetCareWebAPI/DAL/Entities/Appointment.cs
using System.ComponentModel.DataAnnotations;

namespace PetCareWebAPI.DAL.Entities
{
    public class Appointment
    {
        [Key]
        public int IDAppointment { get; set; }
        public DateTime AppointmentDate { get; set; }
        public Adopter Adopter { get; set; } //FK
        public Psichologist Psichologist { get; set; } //FK
    }
}

File: PetCareFront/PetCare/src/app/Entities/appointment.ts
import { Adopter } from "./adopter";
import { Pshicologist } from "./pshicologist";

export interface Appointment {
    idAppointment: string | null;
    appointmentDate: Date;
    adopter: Adopter | null; //FK
    psichologist: Pshicologist | null; //FK
}
